alma/alma-installments-prestashop
# [insurance] Call endpoint with cms references

### Reason for change



[Linear task](https://linear.app/almapay/issue/ECOM-1334/[prestashop]-call-endpoint-with-cms-references)

### Code changes

We add the endpoint sendCustomerCart for track cmsReferences buy

### How to test

Add var_dump in sendCmsReferenceSubscribedForTracking method
Buy insurances
And check if array cmsReferences is sended

### Checklist for authors and reviewers




### Non applicable
